File-Formats/DocIO/convert-markdown-to-word-document-in-csharp.md

```diff
@@ -0,0 +1,576 @@
+    document.Open("Input.md");
+
+    //Save as a Word document to the MemoryStream.
+    MemoryStream outputStream = new MemoryStream();
+    document.Save(outputStream, FormatType.Docx);
+    //Save the stream as a file in the device and invoke it for viewing. 
+    Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("Sample.docx", "application/msword", outputStream);
+    //Please download the helper files from the below link to save the stream as file and open the file for viewing in Xamarin platform
+    //https://help.syncfusion.com/file-formats/docio/create-word-document-in-xamarin#helper-files-for-xamarin
+}
+{% endhighlight %}
+{% endtabs %}
+
+The following code examples show the event handler to customize the image based on source path.
+
+{% tabs %}
+{% highlight c# tabtitle="C#" %}
+private static void MdImportSettings_ImageNodeVisited(object sender, Syncfusion.Office.Markdown.MdImageNodeVisitedEventArgs args)
+{
+    //Set image stream based on image name from input Markdown
+    if (args.Uri == "Image_1.png")
+        args.ImageStream = new FileStream("Image_1.png", FileMode.Open);
+    else if (args.Uri == "Image_2.png")
+        args.ImageStream = new FileStream("Image_2.png", FileMode.Open);
+    //Retrive image from website and use
+    else if (args.Uri.StartsWith("https://"))
+    {
+        WebClient client = new WebClient();
+        //Download the image as stream.
+        byte[] image = client.DownloadData(args.Uri);
+        Stream stream = new MemoryStream(image);
+        //Set the retrieved image from the input Markdown.
+        args.ImageStream = stream;
+    }
+}
+{% endhighlight %}
+
+{% highlight vb.net tabtitle="VB.NET" %}
+Private Shared Sub MdImportSettings_ImageNodeVisited(ByVal sender As Object, ByVal args As Syncfusion.Office.Markdown.MdImageNodeVisitedEventArgs)
+    'Set image stream based on image name from input Markdown
+    If args.Uri Is "Image_1.png" Then
+        args.ImageStream = New FileStream("Image_1.png", FileMode.Open)
+    ElseIf args.Uri Is "Image_2.png" Then
+        args.ImageStream = New FileStream("Image_2.png", FileMode.Open)
+    'Retrive image from website and use
+    ElseIf args.Uri.StartsWith("https://") Then
+        Dim client As WebClient = New WebClient()
+        'Download the image as stream.
+        Dim image As Byte() = client.DownloadData(args.Uri)
+        Dim stream As Stream = New MemoryStream(image)
+        'Set the retrieved image from the input Markdown.
+        args.ImageStream = stream
+    End If
+End Sub
+{% endhighlight %}
+
+{% highlight c# tabtitle="UWP" %}
+private static void MdImportSettings_ImageNodeVisited(object sender, Syncfusion.Office.Markdown.MdImageNodeVisitedEventArgs args)
+{
+    //Set image stream based on image name from input Markdown
+    if (args.Uri == "Image_1.png")
+    {
+        Assembly assembly = typeof(App).GetTypeInfo().Assembly;
+        args.ImageStream = assembly.GetManifestResourceStream("Sample.Assets.Image_1.png");
```

Comment:
Use the sample name 
"Customize_image.Assets.Image_1.png"

Address the same feedbacks for uwp, xamarin